models/marts/marketing/dim_customers_legacy.sql

```diff
@@ -0,0 +1,46 @@
+{{
+    config(
+        transient = false
+    )
+}}
+
+with customer as (
+
+    select * from {{ ref('stg_tpch__customers') }}
+
+),
+nation as (
+
+    select * from {{ ref('stg_tpch__nations') }}
+),
+region as (
+
+    select * from {{ ref('stg_tpch__regions') }}
+
+),
+final as (
+    select 
+        customer.customer_id,
+        customer.name,
+        customer.address,
+        {# nation.nation_id as nation_id, #}
+        nation.name as nation,
+        {# region.region_id as region_id, #}
+        -- region.name as region,
+        case when region.name = 'AFRICA' then 'AFRICA!' else region.name end as region, -- introduce an error with "!"
```

Comment:
Did you want this to error? It looks like you included the value `AFRICA!` in your test so it won't error or warn

Reply:
Yeah, I don't actually want it to error all the time, because then the job fails. I set the severity to warn, but I'll fix the comments!